src/tmplits/tmplits-table/README.md

```diff
@@ -61,23 +61,24 @@ This is a container tmplit for TmplitTableRows. It can be given a title as the f
 Below are the css vars used for styling. These are read into the css properties and are empty by default. If they are null the html uses static default values. See the example on how to populate these variable in an application:
 
 ### Table Header
-| Property | Comment |
-| -------- | ------- |
-| tmplit-table-header-font-size ||
-| tmplit-table-header-font-weight ||
-| tmplit-table-header-padding ||
-| tmplit-table-header-text-align ||
-| tmplit-table-header-color<br>md-sys-color-on-surface ||
-| tmplit-table-header-background-color<br>md-sys-color-surface-container ||
+| Tmplit Token | Material Design Token | Comment |
```

Comment:
When we update LoupeDocs, let's mention how we're falling back to Material tokens.